src/utils/converters.ts

```diff
@@ -28,7 +30,7 @@ const prepareRowsData = (data: PrepareRowsData): transactionProps[] =>
     payee: {
       name: element.payeeName || data.payee.multi,
       // update here the cdn host when avaiable
-      srcImg: element.payeeTaxCode ? `http://cdn.com/${element.payeeTaxCode}.png` : undefined,
+      srcImg: element.payeeTaxCode ? `${CDN_URL}${element.payeeTaxCode}.png` : undefined,
```

Comment:
Setting _undefined_ as fallback will "breaks" the rendering. Maybe it's better to use a "default image"

Reply:
You mean "default image" as a value? Or to actually create a default image and use its url as a fallback? We have a fallback icon for when the src is undefined, so in reality the image won't be rendered at all if the src is undefined.

Reply:
Ok, leave the default fallback icon as actually done